suite-common/test-utils/src/configureMockStore.ts

```diff
@@ -66,6 +66,18 @@ export function configureMockStore<S = any, A extends AnyAction = AnyAction>({
         return next(action);
     });
 
+    const dispatchListeners: ((action: AnyAction) => void)[] = [];
+
+    const onDispatch = (listener: (action: AnyAction) => void) => {
```

Comment:
This is better that the action log, coz I want to do something in sync when this is triggered. Maybe even finish the test or a make a change in state of the store.